### DISCUSSION TITLE
Trouble downloading AS Review - building wheel for Greenlet?

### DISCUSSION THREAD

**Discussion by jackMLW** [asreview/asreview#1687] [Troubleshooting (report errors in the "Issues" tracker)]
Hi. I followed the instructions and installed the latest Miniconda with Python v3.11.5
I use OSMac Monteray v12.1
I then tried the command 'pip install asreview'
However I am getting this error message:

error: command '/usr/bin/clang' failed with exit code 1
            [end of output]
      
        note: This error originates from a subprocess, and is likely not a problem with pip.
        ERROR: Failed building wheel for greenlet
      Failed to build greenlet
      ERROR: Could not build wheels for greenlet, which is required to install pyproject.toml-based projects
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× pip subprocess to install build dependencies did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.

______

Can anyone help please?

**Comment by J535D165** [comment]
This issue is caused by one of the dependencies of asreview: greenlet. 

Can you try to install greenlet with:
```
conda install greenlet
```

And then `pip install asreview` again.

**Reply by jackMLW** [reply]
Thanks, I tried that but I get the same error message, about failure to build wheels for Greenlet:

error: command '/usr/bin/clang' failed with exit code 1
            [end of output]
      
        note: This error originates from a subprocess, and is likely not a problem with pip.
        ERROR: Failed building wheel for greenlet
      Failed to build greenlet
      ERROR: Could not build wheels for greenlet, which is required to install pyproject.toml-based projects
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× pip subprocess to install build dependencies did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.

**Reply by tinakretschmer** [reply]
I have the same problem. Tried installing greenlet but it did not help. Maybe I should add that the problem only occurs on an older Macbook (Monterey) but not on a newer one that's running on Ventura. Could this be the problem?
